deploy/operator/capi/deploy_capi_cluster.sh

```diff
@@ -56,10 +62,53 @@ elif [[ "${IP_STACK}" == "v4v6" ]]; then
     export CLUSTER_HOST_PREFIX_ADDITIONAL="${CLUSTER_HOST_PREFIX_V6}"
     export EXTERNAL_SUBNET_ADDITIONAL="${EXTERNAL_SUBNET_V6}"
     export SERVICE_SUBNET_ADDITIONAL="${SERVICE_SUBNET_V6}"
+    # IPv6 requires hypershift create cluster cidr override
+    export EXTRA_HYPERSHIFT_CREATE_COMMANDS="$EXTRA_HYPERSHIFT_CREATE_COMMANDS --cluster-cidr fd01::/48"
 fi
 
 if [ "${DISCONNECTED}" = "true" ]; then
-    ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE="${LOCAL_REGISTRY}/$(get_image_without_registry ${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE})"
+    export DISCONNECTED_ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE="${LOCAL_REGISTRY}/ocp/ocp-release:latest"
+    # Disconnected hypershift requires:
+    # 1. pull secret in hypershift namespace for the hypershift operator
+    oc get namespace hypershift || oc create namespace hypershift
+    oc get secret "${ASSISTED_PULLSECRET_NAME}" -n hypershift || \
+      oc create secret generic "${ASSISTED_PULLSECRET_NAME}" --from-file=.dockerconfigjson="${ASSISTED_PULLSECRET_JSON}" --type=kubernetes.io/dockerconfigjson -n hypershift
+    # 2. mirrored hypershift operator image to local registry
+    HYPERSHIFT_LOCAL_IMAGE="${LOCAL_REGISTRY}/localimages/hypershift:latest"
+    oc image mirror -a "${PULL_SECRET_FILE}" "${HYPERSHIFT_IMAGE}" "${HYPERSHIFT_LOCAL_IMAGE}"
+    export HYPERSHIFT_IMAGE="${HYPERSHIFT_LOCAL_IMAGE}"
+    # 3. the hypershift cli must be available on the local environment
+    id=$(podman create $HYPERSHIFT_LOCAL_IMAGE)
+    podman cp $id:/usr/bin/hypershift /usr/bin/hypershift
```

Comment:
Can this get us into trouble, each time overwriting the executable? Should it be local in the directory and added to the PATH?

Reply:
Good point! I've modified this as suggested